.github/workflows/test.yaml

```diff
@@ -94,6 +95,15 @@ jobs:
           --exclude_regex "${{ env.KNOWN_FAILURES }}"  \
           --plugin_options "--jit" || true
 
+      - name: Run BPF conformance tests - Cranelift
+        run: |
+          docker run -v ${{github.workspace}}:/rbpf --rm \
+          "${{ env.CONFORMANCE_IMAGE }}" \
+          bin/bpf_conformance_runner --test_file_directory tests \
+          --plugin_path /rbpf/target/release/examples/rbpf_plugin \
+          --exclude_regex "lock"  \
```

Comment:
`mem-len` is passing, so that's why I didn't reuse it. Getting `lock` to work might also not be too much work, but It might be best as a follow up, and then we could remove this entirely.